### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by Pepiloto**

### CODE REVIEW

File: kubernetes/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            name: pwa-frontend
+            port: 
+              number: 8080
+      - pathType: Prefix
+        path: "/api(/*)(.*)"
+        backend:
+          service:
+            name: pwa-backend
+            port: 
+              number: 8081
```
Comment (aurelien-boch) [inline kubernetes/ingress.yaml:32]: will port 8080 & 8081 be exposed to client ? (or are theses ports are only there for IPC purposes)

File: kubernetes/frontend-service.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+kind: Service
+metadata:
+  name: pwa-frontend
+  namespace: pwa
+spec:
+  selector:
+    app: pwa-frontend
+  ports:
+  - port: 3000
+    targetPort: 8080
```
Comment (aurelien-boch) [inline kubernetes/frontend-service.yaml:11]: will the client have to use port 3000 to reach frontend O_o ?

File: kubernetes/backend-service.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+kind: Service
+metadata:
+  name: pwa-backend
+  namespace: pwa
+spec:
+  selector:
+    app: pwa-backend
+  ports:
+  - port: 3000
+    targetPort: 8081
```
Comment (aurelien-boch) [inline kubernetes/backend-service.yaml:11]: why not use 443 for https or 80 for http ?

**Review by aurelien-boch — COMMENTED** [review]
where is the db ?

**Comment by n0no123** [comment]
Le front utilise une variable d'environnement btw :

REACT_APP_API_URL=http://www.example.com/

File: kubernetes/ingress.yaml (hunk 1 above)
Comment (Pepiloto) [reply kubernetes/ingress.yaml]: Client will access the website using the url if that's your question

**Comment by Pepiloto** [comment]
> where is the db ?

On AWS like for Prophecy

**Review by aurelien-boch — APPROVED** [review]

**Pull request merged**